doc/changes/changes_4.2.0.md

```diff
@@ -1,22 +1,34 @@
-# Project Keeper 4.2.0, released 2024-??-??
+# Project Keeper 4.2.0, released 2024-03-11
 
 Code name: Automated Release Process
 
 ## Summary
 
+This release replaces Release Droid workflows `release_droid_.yml` with workflow `release.yml`. This will automatically release the project when `ci-build.yml` succeeded on `main` branch and the changes file contains an up-to-date release date. In case of problems you can start the workflow manually and skip the release to Maven Central or GitHub if necessary.
```

Comment:
```suggestion
This release replaces Release Droid workflows `release_droid_.yml` with workflow `release.yml`. This will automatically release the project when `ci-build.yml` succeeded on `main` branch and the changes file contains an up-to-date release date. In case of problems you can start the workflow manually on GitHub and skip the release to Maven Central or GitHub if necessary.
```